commons/commons-serialization/src/main/kotlin/ac/uk/ebi/biostd/json/serialization/SubmissionJsonSerializer.kt

```diff
@@ -21,11 +20,7 @@ internal class SubmissionJsonSerializer : StdSerializer<Submission>(Submission::
             writeJsonObject(SubFields.SECTION, subm.section)
 
             if (subm is ExtendedSubmission) {
-                writeJsonArray(SubFields.ACCESS_TAGS, merge(subm.accessTags), gen::writeString)
```

Comment:
So, we no longer need these other fields?

Reply:
they were initially added to support exporter calling this system but better simplify until we got to that point and have a proper design.